bibliopixel/drivers/spi_driver_base.py

```diff
@@ -91,4 +174,4 @@ def _compute_packet(self):
 
 def error(text):
     log.error(text)
-    raise IOError(text)
+    raise Exception(text)
```

Comment:
Why this change?  `Exception` is floppy and lacking in specificity. `IOError` is clearer!